js/script.js

```diff
@@ -8,59 +33,183 @@ const formAuthor = document.getElementById('book-author');
 const formCost = document.getElementById('book-cost');
 const formNumberOfCopies = document.getElementById('book-copies');
 
-//Recheck info of EventListener
 form.addEventListener('submit', (e) => {
     e.preventDefault();
 })
-//Make it the way so the table is not being rewritten everytime, but only one row adds to present ones
-function addBook(){
 
-    // Making a new book appear in a dataset of books
-    const idValue = formId.value;
-    const titleValue = formTitle.value;
-    const editionYearValue = formEditionYear.value;
-    const authorValue = formAuthor.value;
-    const costValue = formCost.value;
-    const numberOfCopiesValue = formNumberOfCopies.value;
+function refreshTable() {
+    const tableBody = document.querySelector('.table-container__body');
+    tableBody.innerHTML = '';
+
+    books.forEach((book) => {
+        const row = document.createElement('tr');
+
+        for (let key in book) {
+            const td = document.createElement('td');
+
+            if (key === 'cost') {
+                td.textContent = book[key] + '$';
+                row.appendChild(td);
+            } else {
+                td.textContent = book[key];
+                row.appendChild(td);
+            }
+        }
+        tableBody.appendChild(row);
+        filterTable();
+    })
+}
+refreshTable();

```

Comment:
it would be better to refresh table first time via event listener like https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event